Node/samples/app.ts

```diff
@@ -115,6 +118,23 @@ bot.dialog('FetchMemberList', function (session: builder.Session) {
   );
 });
 
+bot.dialog('FetchTeamInfo', function (session: builder.Session) {
+  var teamId = session.message.sourceEvent.teamsTeamId;
```

Comment:
Let's be consistent and always access the team id as team.id instead of teamsTeamId (and channel id as channel.id instead of teamsChannelId) since that's what we document on MSDN.